debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/echache/LogMinerObjectType.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle.logminer.processor.echache;
+
+public enum LogMinerObjectType {
```

Comment:
Is there a reason for this to be independent of a nested enum in `LogMinerEvent`?  If we do keep this separate, we should add some javadocs here.